src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1046,7 +1063,7 @@ public function is_schedule_time() {
 		}
 
 		// If we are not dealing with the Record Schedule
-		if ( $this->post->post_status !== Tribe__Events__Aggregator__Records::$status->schedule ) {
+		if ( $this->post->post_status !== Records::$status->schedule ) {
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 63efb6e2e4eea5023e6ae15ae8000cc6</sub>